lib/moonbeam-xcm-transactor/XcmTransactorV2.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+// The XcmTransactorV2 contract's address.
+address constant XCM_TRANSACTOR_V2_ADDRESS = 0x000000000000000000000000000000000000080D;
+
+// The XcmTransactorV2 contract's instance.
+XcmTransactorV2 constant XCM_TRANSACTOR_V2_CONTRACT = XcmTransactorV2(
+    XCM_TRANSACTOR_V2_ADDRESS
+);
```

Comment:
What a syntax .... Can you explain what this means? @NunoAlexandre ^^

Reply:
Basically this: 
`<type> constant <constant_name> = <type_constructor>(<type_constructor_arguments>)`

In rust it would be:

``` rust
const XCM_TRANSACTOR_V2_CONTRACT: XcmTransactorV2 = XcmTransactorV2(XCM_TRANSACTOR_V2_ADDRESS);
```